src/views/account-detail/index.js

```diff
@@ -194,48 +194,44 @@ class AccountLatest extends Component {
             let idCounter = 1;
 
             rTxs.forEach(tx => {
-                const events = tx.tx_result.events
-                
-                if (events[1].type === "transfer") {
-                    const attributes = events[1].attributes;
-                    if (attributes[1].key === "amount") {
-                        const value = attributes[1].value.replace("upokt", "");
-
-                        const txHash = tx.hash;
-                        const imageSrc = tx.type.toLowerCase() === "sent" ? sentImgSrc : receivedImgSrc;
-                        const TrClass = document.getElementById("tr-element").className;
-                        const TdClass = document.getElementById("td-element").className;
-
-                        const txTemplate = '<Tr class="' + TrClass + '">\n' +
-                            '<Td class="' + TdClass + '"> <img src=' + imageSrc + ' alt="' + tx.type.toLowerCase() + '" /> </Td>\n' +
-                            '<Td class="' + TdClass + '"> <div class="qty">' + value / 1000000 + ' <span>POKT</span></div> <div class="status">' + tx.type.toLowerCase() + '</div> </Td>\n' +
-                            '<Td class="' + TdClass + ' block-align">' + tx.height + '</Td>\n' +
-                            '<Td class="' + TdClass + '"> <a id="txHashElement' + idCounter + '"> ' + txHash + ' </a> </Td>\n' +
-                            '</Tr>';
-
-                        section.insertAdjacentHTML('beforeend', txTemplate);
-                        // Add onClick event to the clickable element
-                        const toTxDetail = document.getElementById(`txHashElement${idCounter}`);
-
-                        if (toTxDetail) {
-                            toTxDetail.addEventListener("click", () => { this.pushToTxDetail(txHash) });
-                        }
-                        idCounter++;
-                    } else {
-                        console.dir(attributes, { depth: null });
-                    }
-                }
-                
-            })
-            // Display the table
-            this.setState({displayTxListSection: true});
-
-            this.enableLoaderIndicatory(false);
+              if (!tx.stdTx.msg.amount && !tx.stdTx.msg.value) {
```

Comment:
Self review for guidance.

This basically just tells the wallet to not rely on events but rather on the msg property to show the transaction log.